src/javascript/tests/numbers.rs

```diff
@@ -148,3 +148,22 @@ function $divide(a, b) {
 "#
     );
 }
+
+#[test]
+fn integer_patterns() {
+    assert_js!(
+        r#"
+fn go(x) {
+  let 4 = x
+}
+"#,
+        r#""use strict";
+
+function go(x) {
+  let gleam$tmp = x;
+  if (!(gleam$tmp === 4)) throw new Error("Bad match");
+  
```

Comment:
```suggestion
```

Reply:
I think that sounds good but we shouldn't have an empty line at the end of blocks

Reply:
Interesting, I'm not sure how to fix that because the expression code just adds a line to all the lines it generates. is there a way to strip out a line after it's in a document block?